docs/pipeline/configuration.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: "Pipeline Configuration"
+sidebarTitle: "Configurations"
+---
+
+Odigos sets up an observability pipeline in your Kubernetes cluster to collect, process and export OpenTelemetry data.
+
+<Frame>
+    <img src="/images/odigos_pipeline.png" alt="Odigos Pipeline" />
+</Frame>
+
+
+User can configure the following components:
+
+- **Cluster Gateway Collector** - a collector that runs as a k8s Deployment. It receives the OpenTelemetry data from the Node Collectors, processes it, and exports it to the next component in the pipeline.
+
+### Configuring the Pipeline
+
+Odigos offers two main options for configuring the pipeline:
+<Info>
+When configuring the pipeline, you must use **only one of the two methods**—either profiles or directly editing the `odigos-config` ConfigMap. Using both methods will result in configurations overwriting each other, potentially leading to unexpected behavior.
```

Comment:
Can we check before applying the sizing profiles if the config exists and ignore / exit with error?

Reply:
Will add this logic in another PR and their I will also change this message